src/components/Spotlight/Spotlight.tsx

```diff
@@ -275,32 +285,42 @@ const Spotlight = ({
   ]);
 
   // TODO: use a resize observer to detect when the bounds of the target change
-  const updateWindowBounds = () => {
+  const updateWindowBounds = useCallback(() => {
     setWindowDimensions({
       width: window.innerWidth,
       height: window.innerHeight,
     });
-  };
+  }, [setWindowDimensions]);
```

Comment:
Does this do anything? I thought that `setXX` state functions didn't get redefined across renders